.github/workflows/serverless-integration.yml

```diff
@@ -0,0 +1,39 @@
+name: "Serverless Integration Tests"
+
+on:
+  push:
+    paths:
+      - 'cmd/serverless/**'
+      - 'pkg/serverless/**'
+      - 'test/integration/serverless/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout datadog-agent repository
+        uses: actions/checkout@v2
+        with:
+          path: go/src/github.com/DataDog/datadog-agent
```

Comment:
```suggestion
          fetch-depth: 0
          path: go/src/github.com/DataDog/datadog-agent
```

Reply:
Per docs:

> Number of commits to fetch. 0 indicates all history for all branches and tags. Default: 1

I think we want to use the default value (1), no?

Reply:
Got confused by the CodeQL changes, when, in fact, we should fix it there, not here 😅 We can disregard this